spec/eventbrite/event_finder_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "climate_control"
+require "json"
+require "webmock/rspec"
+require "eventbrite/event_finder"
+require "eventbrite/event"
+
+describe Eventbrite::EventFinder do
+  around do |example|
+    env_variables = { EVENTBRITE_ACCESS_TOKEN: "bar", EVENT_ID: "123" }
+
+    ClimateControl.modify env_variables do
+      example.run
+    end
+  end
+
+  describe ".find" do
+    context "when there's a successful response" do
+      it "returns an event object" do
+        allow(Eventbrite::Event).to receive(:new)
+        event_body = { "name" => { "text" => "event info" } }
+        stub_request(:get, eventbrite_api_url).to_return(
+          headers: { "Content-Type" => "application/json" },
+          body: event_body.to_json,
+        )
+
+        Eventbrite::EventFinder.find("123")
+
+        expect(Eventbrite::Event).to have_received(:new).with(event_body)
+      end
+    end
+
+    context "when there's a bad response" do
+      it "returns a placeholder event" do
+        stub_request(:get, eventbrite_api_url).to_return(
+          body: { id: "123" }.to_json,
```

Comment:
Should this have `headers: { "Content-Type" => "application/json" }` too?

Maybe it's worth extracting a `stub_json_get_request` like:

```ruby
def stub_json_get_request(body:)
  stub_request(:get, eventbrite_api_url).to_return(
    headers: { "Content-Type" => "application/json" },
    body: body.to_json,
  )
end
```